Makefile

```
@@ -9,7 +9,7 @@ MODEL_PREFIX ?= ./models/ppo_maze_final
 OUTPUT_GIF ?= evaluation_render.gif
 LEVEL ?= easy
 
-.PHONY: all help train visualize manual install-deps lint format clean
+.PHONY: all help train visualize manual install-deps lint format typecheck clean sync
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)